blocks/sticky-navigation/sticky-navigation.js

```diff
@@ -1,7 +1,7 @@
 import { STICKY_NAVIGATION_DATASET_KEY } from '../../scripts/lib-franklin.js';
 
 function scrollToAnchorWithOffset(anchorId) {
-  const anchorElement = document.getElementById(anchorId);
```

Comment:
getElementById was returning a headings as in this case it had the same id. I changed the logic to be sure to return a section.

Reply:
Actually an ID shall only exist once on a page :/

Reply:
Good point, therefore I changed the logic and made sure the ID is unique so I can keep getElementById.